src/FCQuantities/test_fc_quantities.cpp

```diff
@@ -124,13 +160,23 @@ auto problem_main() -> int {
   sim_write.stopTime_ = 1.0;
   sim_write.maxTimesteps_ = 2e4;
   sim_write.setInitialConditions();
-  sim_write.evolve();
-
-  RadhydroSimulation<FCQuantities> sim_read(BCs_cc);
-  sim_read.cflNumber_ = 0.1;
-  sim_read.stopTime_ = 1.0;
-  sim_read.maxTimesteps_ = 2e4;
-  sim_read.setInitialConditions();
+  amrex::Vector<amrex::Array<amrex::MultiFab, AMREX_SPACEDIM>> const& state_old_fc_write = sim_write.getOldMF_fc();
+  amrex::Vector<amrex::Array<amrex::MultiFab, AMREX_SPACEDIM>> const& state_new_fc_write = sim_write.getNewMF_fc();
+
+  RadhydroSimulation<FCQuantities> sim_restart(BCs_cc);
+  sim_restart.cflNumber_ = 0.1;
+  sim_restart.stopTime_ = 1.0;
+  sim_restart.maxTimesteps_ = 2e4;
```

Comment:
If this just tests the checkpoint files, does this need to run for more than 1 timestep?

Reply:
In fact, does it need to run for any time steps at all? Why not just initialize, write a checkpoint, read it back in, then compare, without ever doing any evolution. That is also safer, because at some point we are going to finish the MHD and then evolution will actually do something, so that the state after evolving for even one time step won't be bitwise identical to the initial state.

Reply:
Yes, agreed. I think this should just remain an I/O test problem. We can create a new test problem for an Alfven wave.

Reply:
Yep, that would be the plan.